test/units/testsuite-74.firstboot.sh

```diff
@@ -24,21 +24,25 @@ ROOT_HASHED_PASSWORD1='$6$foobarsalt$YbwdaATX6IsFxvWbY3QcZj2gB31R/LFRFrjlFrJtTTq
 # shellcheck disable=SC2016
 ROOT_HASHED_PASSWORD2='$6$foobarsalt$q.P2932zYMLbKnjFwIxPI8y3iuxeuJ2BgE372LcZMMnj3Gcg/9mJg2LPKUl.ha0TG/.fRNNnRQcLfzM0SNot3.'
 
+LOCALE_PATH="/etc/locale.conf"
```

Comment:
Could you add a brief comment about that this is Ubuntu (and Debian?) specific.

Reply:
Done, thanks!